<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-DQGIKCA" failures="1" tests="1" name="Test(failed)" time="8.093" errors="0" timestamp="20 May 2022 02:40:31 GMT">
  <testcase classname="Loginpagetest.LandingPageTest" name="Validationhomepagelogo" time="0.318">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Loginpagetest.LandingPageTest.Validationhomepagelogo(LandingPageTest.java:38)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- Validationhomepagelogo -->
  <testcase classname="Loginpagetest.LandingPageTest" name="ValidatetheHomapagetittle" time="0.0">
    <ignored/>
  </testcase> <!-- ValidatetheHomapagetittle -->
  <testcase classname="Loginpagetest.LandingPageTest" name="validatethePresenceofCreateAccount" time="0.0">
    <ignored/>
  </testcase> <!-- validatethePresenceofCreateAccount -->
</testsuite> <!-- Test(failed) -->
